include/zephyr/bluetooth/audio/hap.h

```diff
@@ -0,0 +1,397 @@
+/*
+ * Copyright (c) 2023 Codecoup
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_BLUETOOTH_AUDIO_HAP_H_
+#define ZEPHYR_INCLUDE_BLUETOOTH_AUDIO_HAP_H_
+
+/**
+ * @brief Hearing Access Profile (HAP)
+ *
+ * @defgroup bt_hap Hearing Access Profile (HAP)
+ *
+ * @ingroup bluetooth
+ * @{
+ *
+ * The Hearing Access Profile is used to identify and control a Hearing Aid devices.
+ *
+ * [Experimental] Users should note that the APIs can change as a part of
+ * ongoing development.
+ */
+
+#include <zephyr/bluetooth/bluetooth.h>
+#include <zephyr/bluetooth/audio/has.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/* Hearing Aid Controller Maximum Set Size */
+#define BT_HAP_HARC_SET_SIZE_MAX 2
```

Comment:
HARCs are not part of sets, so this is a confusing name. 

Suggest to just rename to
```suggestion
/* Hearing Aid Controller Maximum Set Size */
#define BT_HAP_SET_SIZE_MAX 2
```